pkg/apis/app/v1alpha1/application_types.go

```diff
@@ -58,6 +58,7 @@ type ApplicationSpec struct {
 	Links []Link `json:"links,omitempty"`
 
 	// Notes contain a human readable snippets intended as a quick start for the users of the Application.
+	// Should contain plain text or markdown-formatted text.
```

Comment:
Can you note using [CommonMark](http://commonmark.org/) as the specific kind and include the version (latest). If you search in the [OpenAPI spec](https://github.com/OAI/OpenAPI-Specification/blob/fb77181150d665cef2257a1f9a5f398aa2404031/versions/3.0.0.md) you can see examples of how they phrased it.

Reply:
Thanks for the link, this is very useful! I'm going to grab some of the wording but 'll drop the RFC-style capitalization.